http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -437,9 +440,18 @@ protected void configure(SelectableChannel selectable)
         });
     }
 
-    private HttpClientTransport getClientTransport(ClientConnector connector, HttpClientConfig config)
+    private HttpClientTransport getClientTransport(ClientConnector connector, SslContextFactory.Client sslContextFactory, HttpClientConfig config)
     {
         ImmutableList.Builder<ClientConnectionFactory.Info> protocols = ImmutableList.builder();
+
+        if (config.isHttp3Enabled()) {
+            ClientQuicConfiguration clientQuicConfig = new ClientQuicConfiguration(sslContextFactory, config.getHttp3PemPath()
+                    .orElseThrow(() -> new RuntimeException("http-client.http3.pem-path needs to be set when http-client.http3.enabled")));
+            HTTP3Client http3Client = new HTTP3Client(clientQuicConfig);
+            http3Client.getQuicConfiguration().setSessionRecvWindow(64 * 1024 * 1024);
```

Comment:
Is this how much memory the client will use? If so, might want to make this configurable